doc/programming/language/types.rst

```diff
@@ -40,6 +40,17 @@ parsed inside a unit.
 - Each ``RANGE`` has one of the forms ``LABEL: A`` or ``LABEL: A..B``
   where ``A`` and ``B`` are bit numbers.
 
+.. rubric:: Constants
+
+- ``bitfield(N) { RANGE_1 [= VALUE_1]; ...; RANGE_N [= VALUE_N] }``
+
+A bitfield constant represents expected values for all or some of the
+individual bitranges. They can be used only for parsing inside a unit
+field, not as values to otherwise operate with. To define such a
+constant with expected values, add ``= VALUE`` to the bitranges inside
+the type definition as suitable. See :ref:`parse_bitfield` for more
+information.
```

Comment:
Maybe a bit of foreshadowing, but could you explicitly spell out that `VALUE` corresponds to the extracted value _after_ applying any `&bit-order` if given (at least that's what I would assume)?

We should we'd also test that.

Reply:
Ack. Also adding a note to #1489 to test that (don't think it'd help much right now where that isn't settled).